source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch, getPRRepositoryInfo} from '../github-helpers';
+
+const isLocalPr = (): boolean => select('.user-select-contain.head-ref a')!.childElementCount === 1;
+
+function checkoutOption(option: string): JSX.Element {
+	const {user, repository} = getPRRepositoryInfo();
+	return (
+		<>
+			{isLocalPr() || <p className="text-gray text-small my-1">{option}</p>}
+			<div className="copyable-terminal">
+				<div className="copyable-terminal-button">
+					<clipboard-copy
+						className="btn btn-sm zeroclipboard-button"
+						role="button"
+						for={`rgh-checkout-pr-${option}`}
+						aria-label="Copy to clipboard"
+						data-copy-feedback="Copied!"
+					>
+						<ClippyIcon/>
+					</clipboard-copy>
+				</div>
+				<pre
+					id={`rgh-checkout-pr-${option}`}
+					className="copyable-terminal-content"
+				>
+					<span className="user-select-contain">
+						{isLocalPr() || `git remote add ${user} ${option === 'HTTPS' ? `${location.origin}/${user}` : `git@${location.hostname}:`}/${repository}.git\n`}
+						git fetch {user} {getCurrentBranch()!}{'\n'}
```

Comment:
On local PRs, it's not `user` but `'origin'`

Reply:
Right 👍 I actually had in mind to change it but JSX was misbehaving... then I realized I had a selling mistake.